crates/consensus/src/receipt/receipts.rs

```diff
@@ -69,7 +69,7 @@ where
     }
 }
 
-impl<T> TxReceipt<T> for Receipt<T>
+impl<T: Clone + fmt::Debug + PartialEq + Eq> TxReceipt<T> for Receipt<T>
```

Comment:
are these now necessary because of the Log type t?

Reply:
Yes exactly